experimental/material_3_demo/lib/home.dart

```diff
@@ -146,15 +153,23 @@ class _HomeState extends State<Home> with SingleTickerProviderStateMixin {
               _ColorSeedButton(
                 handleColorSelect: widget.handleColorSelect,
                 colorSelected: widget.colorSelected,
+                colorSelectionMethod: widget.colorSelectionMethod,
               ),
+              _ColorImageButton(
+                handleImageSelect: widget.handleImageSelect,
+                imageSelected: widget.imageSelected,
+                colorSelectionMethod: widget.colorSelectionMethod,
+              )
             ]
           : [Container()],
     );
   }
 
-  Widget _expandedTrailingActions() => Container(
-        constraints: const BoxConstraints.tightFor(width: 250),
-        padding: const EdgeInsets.symmetric(horizontal: 30),
+  Widget _expandedTrailingActions() {
```

Comment:
Functions like this could be private Widget classes instead. In many cases that means less work to do when this widget rebuilds; about the same otherwise.

Reply:
Thanks - changed.